html/partials/scope_header.html.haml

```diff
@@ -0,0 +1,19 @@
+- meta = @description[scope].meta
+.row
+  .col-xs-1
+    %img.over.scope_logo_big{ |
```

Comment:
Hash attribute should end with one space before the closing brace<br>Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.